src/events/message.rs

```diff
@@ -13,6 +16,22 @@ pub async fn message(ctx: client::Context, msg: Message) -> Result<()> {
         return Ok(());
     }
 
+    if let Some(image) = msg
+        .attachments
+        .iter()
+        .filter(|a| a.height.is_some() && a.height.unwrap() > 0)
+        .next()
+    {
+        let image = image.clone();
+        let id = msg.id.clone();
+        tokio::spawn(async move {
+            if let Ok(resp) = reqwest::get(image.url).await {
+                let mut body = resp.bytes_stream().map_err(|e| serenity::futures::io::Error::new(serenity::futures::io::ErrorKind::Other, e)).into_async_read().compat();
+                tokio::io::copy(&mut body, &mut tokio::fs::File::create(format!("{}/{}", "cache", &id)).await.unwrap()).await.unwrap();
```

Comment:
I'll check the API of reqwest more closely later, this seems,... weird (especially the map_err with the `.unwrap`s, but i assume that's the WIP part :P)

Reply:
turn off your plover_fancytext uwu module, please.